css/pages/index.css

```diff
@@ -2,7 +2,177 @@
   box-sizing: border-box;
 }
 
-.container {
+/* Arnab's code */
+body {
+  margin: 0;
+}
```

Comment:
Kindly leave one line after writing CSS for a class/id. Code will not look crowded that way! 😃

Reply:
We don't comment that it is our code
GitHub lens automatically mentions in whose comments these changes are committed

Reply:
Harshith actually we kept the old css for now we will remove it while integrating all the components of the screen

Reply:
I am mentioning line 5 here

Reply:
> Kindly leave one line after writing CSS for a class/id. Code will not look crowded that way! 😃

I have corrected the code.